envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -47,8 +49,22 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
      * It contains the versions of XDS resources and the contents for a provided node JSON
      * extracted from Envoy's config_dump endpoint.
      */
-    @PostMapping("/snapshot")
-    fun snapshot(@RequestBody node: Node): ResponseEntity<SnapshotDebugInfo> {
+    @PostMapping(value = ["/snapshot"], consumes = ["application/json"], produces = ["application/v2+json"])
```

Comment:
I think this should have `produces = ["application/v2+json", "application/json"]` for compatibility.

Reply:
Done :)